lib/public/UrlChecker.js

```diff
@@ -66,21 +67,25 @@ UrlChecker.prototype.enqueue = function(url, baseUrl, customData)
 	// Undocumented internal use: enqueue(linkObj)
 	if (typeof url==="object" && !(url instanceof String) && url.broken_link_checker===true)
 	{
-		return this.linkQueue.enqueue(
-		{
-			url: url.url.parsed,
-			data: { customData:customData, linkObj:url }
-		});
+		if (shouldQueryUrl(url.url.parsed.href, this)===true) {
```

Comment:
I looked into [jscs](https://npmjs.com/jscs) as a linter. While feature-rich, it's still too limiting for my style which, while consistent in my head, is too complex to be put into a definitive config file. I'd have to contribute greatly to the project to get what I need out of it, unfortunately.